equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/CrmFileLinkedAttachmentMapper.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.laa.crime.equinity.historicaldata.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import uk.gov.justice.laa.crime.equinity.historicaldata.generated.dto.CrmEvidenceFileDTO;
+import uk.gov.justice.laa.crime.equinity.historicaldata.model.CrmEvidenceFileModel;
+
+@Mapper(componentModel = "spring")
+public interface CrmFileLinkedAttachmentMapper extends CrmMapper {
+//    @Mapping(target="files", source="linkedAttachmentsModel")
```

Comment:
commented out code

Reply:
not done yet. commented code might still be used